Add model article

simara-svatopluk opened the pull request.

simara-svatopluk committed:
Add model article

simara-svatopluk commented:
@miryn The first article is waiting for a grammar review that will do my cousin, and I have already finished second. May I ask You for review?

miryn commented:
Hey @simara-svatopluk! I went through the article and it looks better than the previous one. Although the readability has been improved, I had to read some parts more than once.
I also found few grammar mistakes like _"…we are try to imagine…"_ – _"we are trying to imagine"_

Overall, it looks good, I like the idea – keep going! 💪

simara-svatopluk commented:
@miryn Even this article will have a grammar review, but thanks for mentioning... Fixed!

Thanks to Your (and two other guys) suggestions, I added perex, conclusion, references, motivation!, and finally changed one example to a better one.

Can't You see something equally important what I could miss? Any suggestion is great.
Or can You please tell which parts were the most difficult to read?
